### PR TITLE
refactor(button): tertiary button padding

### PR DESCRIPTION
**Pull request by BY00565233**
JIRA: SURF-1961

### LGTM's
* [x] Design LGTM
* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/scss/components/button/mixins/_hxButton.scss [hunk 1]
```diff
@@ -190,8 +190,6 @@
   //
   // Be aware that horizontal padding will affect the appearance of inline tertiary buttons.
   // (e.g., <hx-file-tile> within a drop fence, "retry" button of <hx-file-tile>, etc.)
```
Comment (100stacks) [inline src/scss/components/button/mixins/_hxButton.scss:192]: Please remove this comment block:

```scss
// TODO: remove horizontal padding overrides so as to conform to redline specs
//
// Be aware that horizontal padding will affect the appearance of inline tertiary buttons.
// (e.g., <hx-file-tile> within a drop fence, "retry" button of <hx-file-tile>, etc.)
```

Comment (BY00565233) [reply src/scss/components/button/mixins/_hxButton.scss]: Changes have been done,Please review

File: src/scss/components/button/mixins/_hxButton.scss [hunk 2]
```diff
@@ -186,13 +186,6 @@
   border: 0;
   color: $cyan-900;
 
-  // TODO: remove horizontal padding overrides so as to conform to redline specs
-  //
-  // Be aware that horizontal padding will affect the appearance of inline tertiary buttons.
-  // (e.g., <hx-file-tile> within a drop fence, "retry" button of <hx-file-tile>, etc.)
-  padding-left: 0;
-  padding-right: 0;
```
Comment (100stacks) [inline src/scss/components/button/mixins/_hxButton.scss:194]: @BY00565233, after thinking about this more please add these styles with a `deprecated` class.  Also, include a proper comment that they will be removed in `HelixUI-v1.0.0`.

Comment (BY00565233) [reply src/scss/components/button/mixins/_hxButton.scss:194]: Changes have been done,Please review

File: src/scss/components/button/_button.scss [hunk 3]
```diff
@@ -20,4 +20,10 @@ a[href].hxBtn {
   &.hxTertiary {
     @include hxButton-weight(tertiary);
   }
+
+  // ----- DEPRECATED VARIANTS -----
+  &.hxDeprecated {
+    padding-left: 0;
+    padding-right: 0;
+  }
```
Comment (100stacks) [inline src/scss/components/button/_button.scss:28]: Please verify this is needed.

File: src/scss/components/button/mixins/_hxButton.scss [hunk 4]
```diff
@@ -186,12 +186,11 @@
...
+  // This class will be removed in HelixUI-v1.0.0.
+  .deprecated {
```
Comment (100stacks) [inline src/scss/components/button/mixins/_hxButton.scss:190]: Did we update this to `&.hxDeprecated`?  Please verify.

File: docs/components/button/index.html [hunk 5]
```diff
@@ -263,4 +263,136 @@ <h2 id="button-set">Button Set</h2>
       </footer>
     </div>
   </section>
-{% endblock %}
+
+  {# DEPRECATED BASIC BUTTON #}
+  <section>
+    <header>
+      <h2 id="button-bar">Deprecated Markup - Basic Button</h2>
```
[the remark this answers is not in the document]
Comment (BY00565233) [reply docs/components/button/index.html:270]: Changes have been done,Please review

File: docs/components/button/index.html [hunk 6]
```diff
@@ -263,4 +263,136 @@ <h2 id="button-set">Button Set</h2>
...
+          </a>{% endraw %}
+        {% endcode %}
+      </footer>
+    </div>
+  </section>
+
+  {# DEPRECATED BUTTON BAR #}
+  <section>
+    <header>
+      <h2 id="button-bar">Deprecated Markup - Button Bar</h2>
```
[the remark this answers is not in the document]
Comment (BY00565233) [reply docs/components/button/index.html:350]: Changes have been done,Please review

File: src/scss/components/button/_button.scss (hunk 3 above)
Comment (BY00565233) [reply src/scss/components/button/_button.scss:28]: Yes,It is needed this block

File: src/scss/components/button/mixins/_hxButton.scss (hunk 4 above)
Comment (BY00565233) [reply src/scss/components/button/mixins/_hxButton.scss]: Yes we update this to &.hxDeprecated, and I removed this class in my latest commit.

File: src/scss/components/button/mixins/_hxButton.scss [hunk 7]
```diff
@@ -186,13 +186,6 @@
   border: 0;
   color: $cyan-900;
 
-  // TODO: remove horizontal padding overrides so as to conform to redline specs
-  //
-  // Be aware that horizontal padding will affect the appearance of inline tertiary buttons.
-  // (e.g., <hx-file-tile> within a drop fence, "retry" button of <hx-file-tile>, etc.)
-  padding-left: 0;
```
Comment (100stacks) [inline src/scss/components/button/mixins/_hxButton.scss:193]: Please add:

```scss
 // This class will be removed in HelixUI-v1.0.0.
  &.hxDeprecated {
    padding-left: 0;
    padding-right: 0;
  }
```

Comment (BY00565233) [reply src/scss/components/button/mixins/_hxButton.scss:193]: Changes have been done,Please review

**Pull request merged**